File: HoWo5_6/PhoneCompany/Apple.java
package HoWo5_6.PhoneCompany;

import HoWo5_6.Phone.Smartphone;

public class Apple extends Smartphone {

    public Apple(String model, double sizeDisplay, int akb, String typeOfAntenna, String osVerion, int price) {
        super(model, sizeDisplay, akb, price, ("iOS " + osVerion) , typeOfAntenna);
    }

    @Override
    public String toString() {
        return "Apple " + super.toString();
    }


}

File: HoWo7/PhoneCompany/Apple.py
from Phone.Smartphone import Smartphone


class Apple(Smartphone):

    def __init__(self, model: str, sizeDisplay, akb: int, typeOfAntenna: str, osVerion: str, price: int):
        super().__init__(model, sizeDisplay, akb, price, ("iOS " + osVerion), typeOfAntenna)

    def __str__(self) -> str:
        return "Apple " + super().__str__()